src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHeader.cs

```diff
@@ -16,12 +16,12 @@ namespace System.Formats.Tar
     internal partial struct TarHeader
     {
         // POSIX fields (shared by Ustar and PAX)
-        private const string UstarMagic = "ustar\0";
-        private const string UstarVersion = "00";
+        internal const string UstarMagic = "ustar\0";
```

Comment:
Are these changes from `private` to `internal` still needed? I don't see them used outside of TarHeader with the latest changes.